test/end-to-end/util/helpers.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
will [this solution](https://stackoverflow.com/a/55863857) help us avoid the repeated disable of the eslint rule? we can add the `test` or `test/end-to-end` directory